setup.py

```diff
@@ -17,16 +17,20 @@
 
 import os
 import shutil
+from time import sleep
```

Comment:
It looks like they have the imports organized a bit differently. Perhaps keep "import x" at the top and move the "from x import y" in the next section...all alphabetically, of course.